### PR TITLE
Update Huron scraper

### PR DESCRIPTION
**Pull request by rafe-murray**
The previous dataset, while maintained, does not have the correct roles for the members. As such, I took information from Huron County's site and updated the `roles` and corresponding `label`s to match the language there.

### CODE REVIEW

**Comment by jpmckinney** [comment]
What was the error in the roles?

**Comment by rafe-murray** [comment]
The issue with the roles in the dataset is that all the members are marked as being councillors. As for the existing labels, the Deputy Mayors all used to have the role Councillor (seat 1). I think that they were also labelled as councillors in the previous dataset, but they are now all called Deputy Mayors on the County's site.

**Comment by jpmckinney** [comment]
Hmm, it's a county council composed of the Mayors, Reeves, Deputy Mayors and Deputy Reeves of the municipalities in the county. 

> Biannually, County Council elects a Warden from among the Councillors.

https://www.huroncounty.ca/administration/county-council/

I think this is a similar situation to Grande Prairie County no. 1, so we should just let them all be councillors, and use the CSV.

**Comment by rafe-murray** [comment]
Ok sounds good.

**Comment by rafe-murray** [comment]
I've pushed a different commit which uses the csv, fixing the issue with encoding and doing the seat numbering in a way consistent with other municipalities I've seen (if there is one councillor, there is no seat number added, otherwise every councillor has a seat number added).

File: ca_on_huron/people.py [hunk 1]
```diff
@@ -1,13 +1,29 @@
...
+        single_councillor_districts = ["Howick", "Morris-Turnberry", "North Huron"]
+        people = super().scrape()
+        for p in people:
+            data = p._related[0].post_id.replace("~", "")
+            membership = json.loads(data)
+            for district in single_councillor_districts:
+                if membership["label"] == "{} (seat 1)".format(district):
+                    membership["label"] = district
+                    p._related[0].post_id = "~" + json.dumps(membership)
+            yield p
```
Comment (jpmckinney) [inline ca_on_huron/people.py:29]: Hmm, this is a lot of work just to remove seat numbers. Does it matter?

Comment (rafe-murray) [reply ca_on_huron/people.py:29]: Probably not. I don't think they're used anywhere except in the membership that gets saved

**Pull request merged**